src/cmd/singularity/cli/verify.go

```diff
@@ -17,19 +17,20 @@ import (
 
 func init() {
 	VerifyCmd.Flags().SetInterspersed(false)
+	VerifyCmd.Flags().StringVarP(&keyServerURL, "url", "u", "", "specify the key server URL (default: https://keys.sylabs.io)")
```

Comment:
Suggest same approach here, use the built in default capability of `StringVarP`.